docs/admin/eventing/broker-configuration.md

```diff
@@ -25,27 +25,28 @@ data:
 
 ## Channel implementation options
 
-The following example shows a Broker object where the `spec.config` configuration is specified in a `config-br-default-channel` ConfigMap:
+The following example shows a Broker object where the `spec.config` configuration is specified in a `config-my-broker` ConfigMap:
```

Comment:
I feel like it's more confusing to change this - I understand that you're trying to separate it from channels, but the `config-br-default-channel` ConfigMap gets mentioned below anyway

Reply:
@abrennan89 for posterity: What I find really wrong about this part of the documentation is that we describe how users can use a non-default configuration for their Brokers, but then the example points at... the default ConfigMap. I 100% feel this is missing the point, and eventually we'll have to fix that.